src/simulators.jl

```diff
@@ -166,10 +166,11 @@ end
 
     for step_n in 1:n_steps
         if using_constraints
-            cons_coord_storage .= sys.coords
+            copy!(cons_coord_storage, sys.coords)
```

Comment:
I don't think this makes much difference, if it doesn't it is probably best to retain consistency with the other simulators.